templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,14 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production:
```

Comment:
Do we want this for staging and production? Not sure if there's a better way, but maybe something like:

```yaml
staging: &deploy
  <<: *default
  url:  <%%= ENV.fetch("DATABASE_URL") %>

production:
  <<: *deploy
```